src/game_object.cpp

```diff
@@ -15,7 +15,19 @@ void GameObject::draw(Shader *shader, SceneInfo &scene_info) {
 }
 
 // Updates Game Object Paramters
-void GameObject::update() {        
+void GameObject::update() {
+    // Test code for playing animation for the boy
+    player.set_speed(3.0f);
+
+    if (model->animations.size() > 0) {
+        player.play(model, "walk");
+        /*
+        if (cur_time < 1.5f)
+            player.play(cur_time, model->animations["look"], model);
+        else
+            player.play(cur_time, model->animations["wave"], model);
+        */
```

Comment:
Please make a derived class (like PlayerModel) and override ``GameObject::update`` with it.